src/logic/collectibles/utils/index.ts

```diff
@@ -21,13 +9,7 @@ export const SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH = '42842e0e'
  * @param {string} contractAddress
  * @returns string
  */
-export const getTransferMethodByContractAddress = (contractAddress: string): string => {
-  if (sameAddress(contractAddress, CK_ADDRESS[getNetworkId()])) {
-    // on mainnet `transferFrom` seems to work fine but we can assure that `transfer` will work on both networks
-    // so that's the reason why we're falling back to `transfer` for CryptoKitties
-    return 'transfer'
-  }
-
+export const getTransferMethodByContractAddress = (): string => {
```

Comment:
I remove this as its a hack for Crypto kitties contract which isnt relevent on celo